examples/fishjam-chat/app.json

```diff
@@ -16,12 +16,13 @@
       "bundleIdentifier": "io.fishjam.example.fishjamchat",
       "infoPlist": {
         "NSAppTransportSecurity": {
-          "NSAllowsArbitraryLoads": true //this allows for testing in local network without secured connections
+          "NSAllowsArbitraryLoads": true
         },
         "NSCameraUsageDescription": "We need to access your camera for video calls.",
         "NSMicrophoneUsageDescription": "We need to access your microphone so you can talk during calls.",
         "ITSAppUsesNonExemptEncryption": false
-      }
+      },
+      "appleTeamId": "J5FM626PE2"
```

Comment:
Is this ok to keep it here? 🤔

Reply:
At first I thought it was a good idea, because then we won't have to select team everytime and then I realised that this is a public example 😅